eng/publishPackages/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These shouldn't be necessary anymore, might as well not add them to any of the new projects since it's already gone on `Private.SourceBuilt.Artifacts.csproj`.